stock-plugins/aiverify.stock.fairness-metrics-toolbox-for-classification/algorithms/fairness_metrics_toolbox_for_classification/pytest.ini

```diff
@@ -1,5 +1,4 @@
 # pytest.ini
 [pytest]
-# this is needed to exclude plugin_test.py, which is not a unit test file
-python_files = test_*.py
+testpath=tests
```

Comment:
I get the following warning message when running pytest:
```
PytestConfigWarning: Unknown config option: testpath
```

I think it should by `testpaths`.